jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -413,16 +413,20 @@ public ProtectionDomain protectionDomain(Class<?> clazz) {
 		return clazz.getProtectionDomain();
 	}
 
+	public Object classData(Class<?> clazz) {
+		return null;
+	}
+
 	public MethodHandle stringConcatHelper(String arg0, MethodType type) {
 		return StringConcatHelper.lookupStatic(arg0, type);
 	}
 	
```

Comment:
While you're tidying up, can you remove the trailing tab on this line?

Reply:
Removed it.